src/utils/session-helper.js

```diff
@@ -249,3 +251,36 @@ export const updateTTLFile = async (session, containerUrl, fileObject) => {
     throw new Error('Failed to update ttl file.');
   }
 };
+
+
+
+/**
+ * Creates a TTL file corresponding to an uploaded document or resource
+ *
+ * @memberof utils
+ * @function createResourceTtlFile
+ * @param {fileObjectType} fileObject - Object containing information about file
+ * from form submission (see {@link fileObjectType})
+ * @param {documentUrl} String - url of uploaded document or resource
+ * @returns {object} 
+ */
+export const createResourceTtlFile = async (fileObject, documentUrl) => {
+  let checksum = await createFileChecksum(fileObject);
```

Comment:
For Line 268, I think we can use const. The variable checksum is not being reassigned after being called on with createFileChecksum.

Also, createFileChecksum seem to be defined after createResourceTtlFile. We should move that helper function above this.

Reply:
fixed